### PR TITLE
Add/update CosmosDB examples for native Azure

### PR DESCRIPTION
**Pull request by lblackstone**

### CODE REVIEW

File: azure-nextgen-cs-cosmosdb-logicapp/MyStack.cs [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+        // Create an Azure Resource Group
+        var resourceGroup = new Resources.ResourceGroup("resourceGroup", new Resources.ResourceGroupArgs
+        {
+            ResourceGroupName = "logicappdemo-rg",
+        });
+
+        // Create an Azure resource (Storage Account)
+        var storageAccount = new Storage.StorageAccount("storageAccount", new Storage.StorageAccountArgs
+        {
+            AccountName = "logicappdemosa21",
```
Comment (viveklak) [inline azure-nextgen-cs-cosmosdb-logicapp/MyStack.cs:25]: Nit - with 0.7.0 (autonaming) we should just let the provider handle the names here.

File: azure-nextgen-cs-cosmosdb-logicapp/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+1. Create a new stack:
+
+    ```sh
+    $ pulumi stack init dev
+    ```
+
+2. Set the required configuration variables for this program, and log into Azure:
+
+    ```bash
+    $ pulumi config set azure:location westeurope
```
Comment (viveklak) [inline azure-nextgen-cs-cosmosdb-logicapp/README.md:23]: s/azure/azure-nextgen (or azure-native)

File: azure-nextgen-py-cosmosdb-logicapp/__main__.py [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+import pulumi_azure_nextgen.storage.latest as storage
+import pulumi_azure_nextgen.web.latest as web
+
+# Create an Azure Resource Group
+resource_group = resources.ResourceGroup("resourceGroup", resource_group_name="logicappdemo-rg")
+
+# Create an Azure resource (Storage Account)
+storage_account = storage.StorageAccount(
+    "storageAccount",
+    account_name="logicappdemosa21",
```
Comment (viveklak) [inline azure-nextgen-py-cosmosdb-logicapp/__main__.py:17]: rely on autonaming here with 0.7.0.

File: azure-nextgen-py-cosmosdb-logicapp/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,3 @@
+pulumi>=2.0.0,<3.0.0
+pulumi-azure-nextgen>=0.6.1
```
Comment (viveklak) [inline azure-nextgen-py-cosmosdb-logicapp/requirements.txt:2]: Switch to 0.7.0 (everywhere)

File: azure-nextgen-ts-cosmosdb-logicapp/index.ts [hunk 5]
```diff
@@ -55,24 +56,19 @@ const dbContainer = new documentdb.SqlResourceSqlContainer("container", {
...
 });
 
-const accountKeys = pulumi.all([resourceGroup.name, cosmosdbAccount.name]).apply(async args => {
-    const resourceGroupName = args[0];
-    const accountName = args[1];
-    const accountKeys = documentdb.listDatabaseAccountKeys({
-        accountName: accountName,
+const accountKeys = pulumi
+    .all([cosmosdbAccount.name, resourceGroup.name])
+    .apply(([cosmosdbAccountName, resourceGroupName]) => documentdb.listDatabaseAccountKeys({
```
Comment (viveklak) [inline azure-nextgen-ts-cosmosdb-logicapp/index.ts:61]: Thanks! I was trying to find a neat way to unfold the list but punted :)

File: azure-nextgen-py-cosmosdb-logicapp/README.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+1. Create a new stack:
+
+    ```sh
+    $ pulumi stack init dev
+    ```
+
+2. Set the required configuration variables for this program, and log into Azure:
+
+    ```bash
+    $ pulumi config set azure:location westeurope
```
Comment (viveklak) [inline azure-nextgen-py-cosmosdb-logicapp/README.md:24]:
```suggestion
    $ pulumi config set azure-nextgen:location westeurope
```

**Comment by viveklak** [comment]
Looks good overall. We can either merge with the existing azure-nextgen dependency (probably still want to fix the readme's config entries for setting location) and follow-up with a conversion to azure-native or do all of that now. Whatever you prefer @lblackstone

**Review by viveklak — APPROVED** [review]

**Comment by lblackstone** [comment]
> Looks good overall. We can either merge with the existing azure-nextgen dependency (probably still want to fix the readme's config entries for setting location) and follow-up with a conversion to azure-native or do all of that now. Whatever you prefer @lblackstone

I'll update to 0.7.0 on this PR.

**Comment by mikhailshilkov** [comment]
I'll merge this and open an azure-native PR for all examples

**Pull request merged**